k8s_handle/k8s/api_extensions.py

```diff
@@ -42,3 +42,40 @@ def list_api_resource_arbitrary(self, group, version):
 
             log.error('{}'.format(add_indent(e.body)))
             raise ProvisioningError(e)
+
+
+class CoreResourcesAPI(client.CoreApi):
+
+    def list_api_resources(self, version):
+        try:
+            return self.api_client.call_api(
```

Comment:
Why we set all parameters for this api call? What reason? Most of parameters have default values.